src/main/java/net/sf/jabref/logic/formatter/bibtexfields/UnicodeToLatexFormatter.java

```diff
@@ -39,10 +39,9 @@ public String format(String text) {
         }
 
         // Standard symbols
-        Set<Character> chars = HTMLUnicodeConversionMaps.UNICODE_LATEX_CONVERSION_MAP.keySet();
-        for (Character character : chars) {
-            result = result.replace(character.toString(),
-                    HTMLUnicodeConversionMaps.UNICODE_LATEX_CONVERSION_MAP.get(character));
+        Set<String> strings = HTMLUnicodeConversionMaps.UNICODE_LATEX_CONVERSION_MAP.keySet();
```

Comment:
Please find a more speaking variable name  as "strings" here and below in the loop
e.g unicode_strings or whatever..